metadata-models/src/main/pegasus/com/linkedin/metadata/key/TelemetryKey.pdl

```diff
@@ -0,0 +1,14 @@
+namespace com.linkedin.metadata.key
+
+/**
+ * Key for a Tag
```

Comment:
Can we add documentation saying that there should only be one key?

Reply:
Yep!

Reply:
This has a copy & paste error anyway, thanks for pointing it out 😄